### PR TITLE
User past order view; closes #86

### LINKED ISSUE #86: A logged in user, should be able to see past orders

### PR DESCRIPTION
**Pull request by EmilyMB**
Allows users to see their past orders (but not anyone else's). Redirects guests to root when attempting to see visit the past orders page.

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -12,6 +12,6 @@ def formatted_created_at
   end
 
   def total_dollar_amount
-    number_to_currency(total_price / 100)
+    number_to_currency(total_price / 100.00)
```
Comment (mikedao) [inline app/models/order.rb:15]: Forcing a cents column, right?

File: spec/integration/guest_view_spec.rb [hunk 2]
```diff
@@ -84,6 +85,14 @@
...
+    it "redirects the guest to the root when attempting to visit orders_path" do
+      visit orders_path
+
+      expect(current_path).to eq(root_path)
+    end
+  end
+
   def create_item
     image = create(:image)
     item = FactoryGirl.create(:item, title: "milk",
```
Comment (mikedao) [inline spec/integration/guest_view_spec.rb]: You shouldnt need FactoryGirl  here, you can just call create, did you merge in latest master? I think i changed this in a previous commit while cleaning out the test files.

File: app/models/order.rb (hunk 1 above)
Comment (EmilyMB) [inline app/models/order.rb:15]: Yes, I think there are other places that we need to do that as well (we really should pull all of them out into a helper method).

File: spec/integration/guest_view_spec.rb (hunk 2 above)
Comment (EmilyMB) [inline spec/integration/guest_view_spec.rb]: Yes, I did merge. Let me try again.

File: app/models/order.rb (hunk 1 above)
Comment (EmilyMB) [inline app/models/order.rb:15]: I'll open a Waffle ticket so we don't forget.

**Comment by mikedao** [comment]
:+1:
